core/lib/zksync_core/src/api_server/tx_sender/mod.rs

```diff
@@ -88,7 +88,9 @@ impl MultiVMBaseSystemContracts {
             | ProtocolVersionId::Version15
             | ProtocolVersionId::Version16
             | ProtocolVersionId::Version17 => self.post_virtual_blocks_finish_upgrade_fix,
-            ProtocolVersionId::Version18 | ProtocolVersionId::Version19 => self.post_boojum,
+            ProtocolVersionId::Version18
+            | ProtocolVersionId::Version19
+            | ProtocolVersionId::Version20 => self.post_boojum,
```

Comment:
Bootloader was changed, so you need to branch here

Reply:
@perekopskiy its interface didn't, so I believe we can keep the old one

Reply:
But gas costs have been slightly changed.. I'm not sure if it affects API responses though. @Deniallugo what do you think?

Reply:
@perekopskiy it does affect them (for estimateGas), I just thought that we generally try to avoid adding new bootloaders if we can avoid doing it, but I don't mind. I'll add one